react/features/toolbox/components/native/OverflowMenu.js

```diff
@@ -0,0 +1,91 @@
+// @flow
+
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import { hideDialog, BottomSheet } from '../../../base/dialog';
+import { AudioRouteButton } from '../../../mobile/audio-mode';
+import { PictureInPictureButton } from '../../../mobile/picture-in-picture';
+import { RoomLockButton } from '../../../room-lock';
+
+import AudioOnlyButton from './AudioOnlyButton';
+import ToggleCameraButton from './ToggleCameraButton';
+
+import { overflowMenuItemStyles } from './styles';
+
+type Props = {
+
+    /**
+     * Used for hiding the dialog when the selection was completed.
+     */
+    dispatch: Function,
+};
+
+/**
+ * The exported React {@code Component}. We need a reference to the wrapped
+ * component in order to be able to hide it using the dialog hiding logic.
+ */
+
+// eslint-disable-next-line prefer-const
+let OverflowMenu_;
+
+/**
+ * Implements a React {@code Component} with some extra actions in addition to
+ * those in the toolbar.
+ */
+class OverflowMenu extends Component<Props> {
+    /**
+     * Initializes a new {@code OverflowMenu} instance.
+     *
+     * @inheritdoc
+     */
+    constructor(props: Props) {
+        super(props);
+
+        this._onCancel = this._onCancel.bind(this);
+    }
+
+    /**
+     * Implements React's {@link Component#render()}.
+     *
+     * @inheritdoc
+     * @returns {ReactElement}
+     */
+    render() {
+        return (
+            <BottomSheet onCancel = { this._onCancel }>
+                <AudioRouteButton
+                    showLabel = { true }
+                    styles = { overflowMenuItemStyles } />
+                <ToggleCameraButton
+                    showLabel = { true }
+                    styles = { overflowMenuItemStyles } />
+                <AudioOnlyButton
+                    showLabel = { true }
+                    styles = { overflowMenuItemStyles } />
+                <RoomLockButton
+                    showLabel = { true }
+                    styles = { overflowMenuItemStyles } />
+                <PictureInPictureButton
+                    showLabel = { true }
+                    styles = { overflowMenuItemStyles } />
+            </BottomSheet>
+        );
+    }
+
+    _onCancel: () => void;
+
+    /**
+     * Hides the dialog.
+     *
+     * @private
+     * @returns {void}
+     */
+    _onCancel() {
+        this.props.dispatch(hideDialog(OverflowMenu_));
```

Comment:
@saghul, this comment is not a request for action on this PR: I'm not sure whether we should better dismiss the overflow menu on item tap. For example, I see the iOS share sheet dismiss itself on item tap.

Reply:
Yeah, I was unsure about what to do there. I can give it a shot and we can see how it feels. That like you commented on, however, needs to stay, because the sheet also needs to hide when tapping outside, right?

Reply:
> That like you commented on, however, needs to stay, because the sheet also needs to hide when tapping outside, right?

Right. I wasn't commenting on the line itself, I merely grabbed a related piece of source code and put my additional thoughts.